### DISCUSSION TITLE
Add app shortcuts

### DISCUSSION THREAD

**Discussion by kilotaras** [iSoron/uhabits#393] [Feature Requests]
Android 7.1 introduced app shortcuts, which can be used from long press or even searched from some launchers (e.g Nova + Sesame). They can also be added to home screen and, unlike widgets, be put in folder there.

https://developer.android.com/guide/topics/ui/shortcuts.html

Please consider adding shortcuts for habits.

**Comment by iSoron** [comment]
I am aware of this new Android feature, but I'm not sure how could Loop benefit from it. What kind of app shortcuts do you think it would be interesting?

**Reply by VishnuSanal** [reply]
> What kind of app shortcuts do you think it would be interesting?

I think top habits can be added as dynamic shortcuts, which when clicked leads to the details screen of the respective habit.

What do you think? If you think we can have this feature on Loop, I'd like to work on it.

**Reply by VishnuSanal** [reply]
> > What kind of app shortcuts do you think it would be interesting?
> 
> I think top habits can be added as dynamic shortcuts, which when clicked leads to the details screen of the respective habit.
> 
> What do you think? If you think we can have this feature on Loop, I'd like to work on it.

Hi @iSoron! What do you think? (:
